control/control.go

```diff
@@ -63,7 +45,9 @@ func (c *Controller) Register(server *grpc.Server) error {
 }
 
 func (c *Controller) DiskUsage(ctx context.Context, r *controlapi.DiskUsageRequest) (*controlapi.DiskUsageResponse, error) {
-	du, err := c.opt.CacheManager.DiskUsage(ctx, client.DiskUsageInfo{
+	// FIXME mw0
+	mw0 := c.opt.MetaWorkers[0]
```

Comment:
I think this is probably what the `TODO` is for but it should iterate over the workers and add worker ID to the `DiskUsageInfo` (can be done is separate PR).